analyst/workspace/1_import_bagfiles.ipynb

```diff
@@ -23,16 +23,99 @@
+      "Topics found in data/bags/2024-03-21_tim/bsharp/Fixed/20240321_2243_survey_bsharp2_0.fix_all.bag:\n",
+      "{'/hw/depth_haz/points': 271, '/hw/depth_haz/extended/amplitude_int': 271, '/gnc/ekf': 4810, '/hw/imu': 6270, '/loc/pose': 4800, '/loc/ml/features': 72, '/graph_loc/state': 391, '/sparse_mapping/pose': 69, '/loc/of/features': 417, '/mob/flight_mode': 1, '/hw/wifi': 547, '/hw/depth_haz/extended': 271, '/mgt/img_sampler/nav_cam/image_record': 254, '/hw/cam_nav/image_color': 254, '/hw/cam_nav_bayer': 254, '/beh/inspection/status': 272, '/hw/cam_sci_info': 109, '/tf': 4797, '/hw/cam_nav_ctrl': 55, '/mgt/sys_monitor/time_sync': 108}\n",
+      "\n",
+      "Topics found in data/bags/2024-03-21_tim/bsharp/Fixed/20240321_2252_survey_bsharp6_0.fix_all.bag:\n",
+      "{'/hw/depth_haz/points': 272, '/hw/depth_haz/extended/amplitude_int': 272, '/gnc/ekf': 4921, '/loc/pose': 4906, '/hw/imu': 6374, '/loc/of/features': 448, '/mob/flight_mode': 1, '/graph_loc/state': 422, '/hw/cam_nav_ctrl': 56, '/loc/ml/features': 67, '/hw/wifi': 550, '/sparse_mapping/pose': 66, '/mgt/img_sampler/nav_cam/image_record': 254, '/hw/cam_nav/image_color': 254, '/hw/cam_nav_bayer': 254, '/hw/depth_haz/extended': 272, '/mgt/sys_monitor/time_sync': 110, '/beh/inspection/status': 275, '/tf': 4897, '/hw/cam_sci_info': 42}\n",
+      "Reading  25000 / 25478   25000 / 27147  \n",
+      "Topics found in data/bags/2024-03-21_tim/bsharp/Fixed/20240321_2248_survey_bsharp3_0.fix_all.bag:\n",
+      "{'/hw/depth_haz/points': 278, '/hw/depth_haz/extended/amplitude_int': 278, '/gnc/ekf': 5082, '/hw/imu': 6611, '/loc/pose': 5065, '/loc/ml/features': 73, '/loc/of/features': 447, '/mob/flight_mode': 1, '/graph_loc/state': 446, '/sparse_mapping/pose': 70, '/hw/wifi': 556, '/mgt/img_sampler/nav_cam/image_record': 248, '/hw/cam_nav/image_color': 248, '/hw/cam_nav_bayer': 248, '/hw/depth_haz/extended': 278, '/beh/inspection/status': 279, '/mgt/sys_monitor/time_sync': 111, '/tf': 5065, '/hw/cam_nav_ctrl': 56, '/hw/cam_sci_info': 38}\n",
+      "Reading  26000 / 27147   27000 / 27147  \n",
+      "Topics found in data/bags/2024-03-21_tim/bsharp/Fixed/20240321_2254_survey_bsharp7_0.fix_all.bag:\n",
+      "{'/hw/depth_haz/points': 297, '/hw/depth_haz/extended/amplitude_int': 297, '/gnc/ekf': 5381, '/hw/imu': 6996, '/loc/pose': 5363, '/loc/of/features': 488, '/graph_loc/state': 503, '/sparse_mapping/pose': 85, '/mob/flight_mode': 1, '/hw/wifi': 606, '/mgt/img_sampler/nav_cam/image_record': 255, '/hw/cam_nav/image_color': 255, '/hw/cam_nav_bayer': 255, '/hw/cam_nav_ctrl': 61, '/hw/depth_haz/extended': 297, '/beh/inspection/status': 303, '/hw/cam_sci_info': 121, '/tf': 5379, '/loc/ml/features': 84, '/mgt/sys_monitor/time_sync': 120}\n",
+      "Done loading database\n"
+     ]
+    }
+   ],
    "source": [
     "from scripts.load_bag_database import LoadBagDatabase\n",
-    "path=\"data/bags/\"\n",
+    "path=\"data/bags/2024-03-21_tim/bsharp/Fixed\"\n",
```

Comment:
not sure you want to commit the changes to this file. They seem specific to you.